core-services/egov-indexer/src/main/java/org/egov/infra/indexer/util/IndexerUtils.java

```diff
@@ -47,6 +68,7 @@ public class IndexerUtils {
 	private RestTemplate restTemplate;
 
 	@Autowired
+	@Lazy
```

Comment:
Lazy initialization to bypass circular dependencies won't help as cyclic dependencies still pose a threat in terms of potential memory leaks as managed objects might not be released properly due to cyclic references.